content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-probes.md

```diff
@@ -309,6 +309,9 @@ Here's one scenario where you would set it. Suppose the Container listens on 127
 and the Pod's `hostNetwork` field is true. Then `host`, under `httpGet`, should be set
 to 127.0.0.1. If your pod relies on virtual hosts, which is probably the more common
 case, you should not use `host`, but rather set the `Host` header in `httpHeaders`.
+kubelet will make the probe connection at the node but not in the pod which would want
```

Comment:
For me, I'm afraid I'm finding this sentence hard to read. I already know what's meant; I'm concerned that readers who don't might struggle to get a clear understanding. Maybe a more verbose explanation in its own paragraph would work better.

Reply:
please review again, thanks.